appendices/rdp-controller.md

```diff
@@ -0,0 +1,35 @@
+# RDP Controller
+
+RDP stands for Remote Desktop Protocol and is the foundation for remote desktop connections in Windows. With RDP you can connect to a remote computer running Windows and use it as if it was your local machine.
+
+Our RDP Controller is a tool for running G1ANT.Robot on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection. The script will continue the execution uninterrupted.
```

Comment:
Try something like that:
"Our RDP Controller is a tool allows running automations with G1ANT.Robot on a remote PC without disturbance even when you close you RDP client and disconnect RDP session..."

Reply:
The sense is almost the same, only my description is more detailed and probably more comprehensive for people who are not that advanced in IT.

Reply:
```suggestion
Our RDP Controller is a tool for running G1ANT.Robot automations on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection client. The script will continue the execution without disturbance.
```